test/unit/helpers/recipe_helper_test.rb

```diff
@@ -0,0 +1,19 @@
+require "test_helper"
+
+class RecipeHelperTest < ActionView::TestCase
+  include ERB::Util
```

Comment:
Without this, the tests were blowing up with `h method not in class`. This could totally be pilot error on how I setup this test.